modules/@apostrophecms/piece-type/index.js

```diff
@@ -839,7 +834,6 @@ module.exports = {
         // Options specific to pieces and their manage modal
         browserOptions.filters = self.filters;
         browserOptions.columns = self.columns;
-        browserOptions.contextual = self.contextual;
```

Comment:
this stuff seemed to be driving the context menu options.

Reply:
I would mark this one with a TODO instead as the UI does need to know if we decide to keep the idea of a configuration flag for whether to redirect on create.